torch/_dynamo/variables/torch.py

```diff
@@ -721,6 +642,150 @@ def check_any_unspec(x):
 
             return tensor_variable
 
+    def _call_ntuple(self, tx, args, kwargs):
+        """inline behavior of torch.nn.modules.utils._ntuple"""
+        if self.value is torch.nn.modules.utils._ntuple:
+            count = args[0].as_python_constant()
+        else:
+            count = self.value.__closure__[0].cell_contents
+        assert isinstance(count, int)
+        assert not kwargs
+
+        def handle_ntuple(value):
+            if value.has_unpack_var_sequence(tx):
+                return variables.TupleVariable(
+                    list(value.unpack_var_sequence(tx)),
+                )
+            elif value.is_python_constant():
+                # constant prop through it
+                return variables.ConstantVariable.create(
+                    torch.nn.modules.utils._ntuple(count)(value.as_python_constant()),
+                )
+            else:
+                unimplemented(f"torch.nn.modules.utils._ntuple({value})")
+
+        if self.value is torch.nn.modules.utils._ntuple:
+            return variables.LambdaVariable(handle_ntuple)
+        else:
+            return handle_ntuple(args[0])
+
+
+class TorchVariable(BaseTorchVariable):
+    """Points to a module, classes or functions in torch.*"""
+
+    def __init__(self, value, **kwargs):
+        if (
+            isinstance(value, collections.abc.Hashable)
+            and value in tensor_dunder_fns_remap
+        ):
+            value = tensor_dunder_fns_remap[value]
```

Comment:
Is there a reason you don't do this in the super constructor

Reply:
I think this is not needed any more, so I leave as is.

Reply:
Double checked, this is not needed any more, we can remove it later.